build/root/var/lib/ci-artifacts/entrypoints.d/gpu-operator-run-libs.sh

```diff
@@ -0,0 +1,65 @@
+prepare_cluster_for_gpu_operator() {
+    toolbox/cluster/capture_environment.sh
+
+    entitle
+
+    if ! toolbox/nfd/has_nfd_labels.sh; then
+        toolbox/nfd/deploy_from_operatorhub.sh
+    fi
+
+    if ! toolbox/nfd/has_gpu_nodes.sh; then
+        toolbox/cluster/scaleup.sh
+        toolbox/nfd/wait_gpu_nodes.sh
+    fi
+}
+
+validate_gpu_operator_deployment() {
+    trap toolbox/gpu-operator/capture_deployment_state.sh ERR
+
+    toolbox/gpu-operator/wait_deployment.sh
+    toolbox/gpu-operator/run_gpu_burn.sh
+    toolbox/gpu-operator/capture_deployment_state.sh
+}
+
+test-master-branch() {
```

Comment:
Jiri pointed out that `-` are not explicitely allowed in `bash` (but supported), and forbidden in `sh`

Reply:
Should we replace all - with _?

Reply:
yes please